packages/core/src/tracing/index.js

```diff
@@ -15,9 +15,7 @@ const tracingUtil = require('./tracingUtil');
 const spanBuffer = require('./spanBuffer');
 const supportedVersion = require('./supportedVersion');
 const { otelInstrumentations } = require('./opentelemetry-instrumentations');
-const { isLatestEsmSupportedVersion, hasExperimentalLoaderFlag, isESMApp } = require('../util/esm');
-const iitmHook = require('../util/iitmHook');
-const { getPreloadFlags } = require('../util/getPreloadFlags');
+const coreUtil = require('../util');
```

Comment:
We have to make clear that these are the core utils.
They will get initialized in the core/src/index module in #1556 
But they can be used & activated anywhere.
There is always more room for further optimizations later.

e.g. We should rename `getPreloadFlags` to simply `preloadFlags`. But it will create a large conflict between the pull requests.